src/webview/components/Molecules/Tabs/Tabs.tsx

```diff
@@ -6,7 +6,7 @@ interface TabProps {
   children: React.ReactNode[];
 }
 
-const Tabs: React.FC<TabProps> = ({ headers, children }) => {
```

Comment:
Recommend not using React.FC anymore I was looking into it and its fully redundant in modern React now 😭